integration/presto/src/main/prestodb/org/apache/carbondata/presto/CarbondataSplitManager.java

```diff
@@ -117,6 +122,16 @@ public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHand
       // file metastore case tablePath can be null, so get from location
       location = table.getStorage().getLocation();
     }
+    List<PartitionSpec> filteredPartitions = new ArrayList<>();
```

Comment:
Can you add a testcase with partition filter?

Reply:
please read the description, I have mentioned why UT cannot be added now